hawkmoth/__init__.py

```diff
@@ -162,21 +173,28 @@ class CAutoDocDirective(CAutoBaseDirective):
     optional_arguments = 100   # arbitrary limit
 
     def _get_filenames(self):
+        filenames = []
+
         for pattern in self.arguments:
-            filenames = glob.glob(self.env.config.cautodoc_root + '/' + pattern)
-            if len(filenames) == 0:
+            candidates = glob.glob(os.path.join(self.env.config.cautodoc_root, pattern))
```

Comment:
Fixes like using os.path.join() should be done and merged first.

Reply:
True, I'll split it up. Thanks for the comments by the way.